src/js/tech/html5.js

```diff
@@ -549,9 +549,19 @@ class Html5 extends Tech {
    * @method setSrc
    */
   setSrc(src) {
+    let loadstartlistener = Html5.prototype.loadStartListener_;
+
+    this.off(this.el_, 'loadstart', loadstartlistener);
+    this.one(this.el_, 'loadstart', () => this.one(this.el_, 'loadstart', loadstartlistener));
```

Comment:
Yeah, this doesn't account for `setSource` which is what ends up being used if a source handler is in effect.
https://github.com/videojs/video.js/blob/2e2dbde4b4b70ecac22736477ca60d20a12cfadd/src/js/player.js#L1916-L1919